### PR TITLE
🗑️ Use AppleScript instead of Swift for trashing.

### PR DESCRIPTION
**Pull request by reitermarkus**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [x] Have you successfully run `brew tests` with your changes locally?

### CODE REVIEW

File: Library/Homebrew/cask/lib/hbc/artifact/uninstall_base.rb [hunk 1]
```diff
@@ -226,12 +228,35 @@ def uninstall_delete(*paths)
       def uninstall_trash(*paths)
         return if paths.empty?
 
+        return `say No trash for you!` if Utils.current_user == "ilovezfs"
```
Comment (ilovezfs) [inline Library/Homebrew/cask/lib/hbc/artifact/uninstall_base.rb]: LOL. :rofl:

Comment (dunn) [inline Library/Homebrew/cask/lib/hbc/artifact/uninstall_base.rb]: 👍

**Comment by ilovezfs** [comment]
>No easter egg for you!

:sob:

File: Library/Homebrew/cask/lib/hbc/artifact/uninstall_base.rb [hunk 2]
```diff
@@ -226,12 +228,33 @@ def uninstall_delete(*paths)
...
+
+              repeat with i from 1 to (count trashedItems)
+                set item i of trashedItems to POSIX path of (item i of trashedItems as string)
+                set output to output & (item i of trashedItems) & (do shell script "printf \"\\0\"")
+              end repeat
+
+              return output
+            end tell
+          end run
+        EOS
```
Comment (alyssais) [inline Library/Homebrew/cask/lib/hbc/artifact/uninstall_base.rb]: Could we loop in Ruby instead to keep AppleScript to a minimum?

Comment (reitermarkus) [inline Library/Homebrew/cask/lib/hbc/artifact/uninstall_base.rb]: No, otherwise it will make the trashing sound for every single file.

Comment (ilovezfs) [inline Library/Homebrew/cask/lib/hbc/artifact/uninstall_base.rb]: That could be considered a feature.

Comment (vitorgalvao) [inline Library/Homebrew/cask/lib/hbc/artifact/uninstall_base.rb]: Also note that in addition to having the trashing sound happen multiple times (which can be funny: it’s so fast it sounds like a paper machine gun), the Finder’s undo would only work for one file at a time (and to a limit). This way, doing undo once will recover everything to their right places.

Comment (alyssais) [inline Library/Homebrew/cask/lib/hbc/artifact/uninstall_base.rb]: Awesome that it hooks into undo!

**Pull request merged**